packages/manager/modules/user-contracts/src/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Maybe we can try to get translations from existing modules if they already exist

Reply:
Have added but we may have new text change, ill check with the requirement